deploymentTemplates/aks-hci/azuredeploy.json

```diff
@@ -0,0 +1,170 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string"
+    },
+    "aksClusterName": {
+      "type": "string"
+    },
+    "aksAdminGroupObjectId": {
+      "type": "string",
+      "defaultValue": ""
+    },
+    "aksControlPlaneIP": {
+      "type": "string"
+    },
+    "aksControlPlaneNodeCount": {
+      "type": "int",
+      "defaultValue": 1,
+      "maxValue": 5
+    },
+    "aksControlPlaneNodeSize": {
+      "type": "string",
+      "defaultValue": "Standard_A4_v2"
+    },
+    "aksPodCidr": {
+      "type": "string",
+      "defaultValue": "10.244.0.0/16"
+    },
+    "aksKubernetesVersion": {
+      "type": "string",
+      "defaultValue": "v1.26.6"
```

Comment:
suggest leave the kubernetes version blank/empty string so user can specify required version

Reply:
That was just a default value in case they forgot to specify but did remove it.